### PR TITLE
WIP DO NOT MERGE Update min vid description to include audio support

### PR DESCRIPTION
**Pull request by meandavejustice**
cc/ @johngruen for screenshots.
We'll probably want:
* Audio playback (wave view)
* Audio playback (frequency view)

You can click on the player's timestamp to switch audio visualization view.

Would we also like to adjust tour steps in this pr? (they are probably fine as is)

### CODE REVIEW

**Comment by flodolo** [comment]
I know it's a WIP, but just a reminder that the final PR will need to include the change to .ftl file.

File: content-src/experiments/min-vid.yaml [hunk 1]
```diff
@@ -3,8 +3,9 @@ title: 'Min Vid'
 slug: min-vid
 thumbnail: /static/images/experiments/min-vid/experiments_experiment/thumbnail.png
 description: >
-  Keep videos front and center. Min Vid lets you display YouTube and Vimeo
-  videos in a small frame that stays in the foreground as you browse the web.
+  Take control of media on the web. Display YouTube, Vimeo, Soundcloud
```
Comment (pdehaan) [inline content-src/experiments/min-vid.yaml]: nit: SoundCloud (per https://on.soundcloud.com/)

**Comment by meandavejustice** [comment]
@johngruen Will you be available to add screenshots this week?

**Comment by jaredhirsch** [comment]
This PR is looking pretty stale. @meandavejustice do you still need screenshots? looks like yes, based on the TxP prod site?

**Comment by meandavejustice** [comment]
Will reopen when queue support is complete

**Pull request closed without merging**